### PR TITLE
cmd: oci-image-tool->oci-xxxx

### PR DESCRIPTION
**Pull request by TrumanLing**
oci-image-tool->oci-create-runtime-bundle
oci-image-tool->oci-unpack
oci-image-tool->oci-image-validate

Signed-off-by: Ling FaKe <lingfake@huawei.com>

### CODE REVIEW

File: cmd/oci-create-runtime-bundle/main.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 	cmd.Flags().StringVar(
 		&v.typ, "type", "",
 		fmt.Sprintf(
-			`Type of the file to unpack. If unset, oci-image-tool will try to auto-detect the type. One of "%s"`,
+			`Type of the file to be unpacked. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "%s"`,
```
Comment (wking) [inline cmd/oci-create-runtime-bundle/main.go]: I don't think we need the “unpack” → “be unpacked”.

The `oci-image-tool` → `oci-*` changes look good.

**Comment by TrumanLing** [comment]
@wking updated it, PTAL.

**Comment by wking** [comment]
415eb62 looks good to me.

**Comment by xiekeyang** [comment]
LGTM

**Comment by coolljt0725** [comment]
LGTM

**Comment by xiekeyang** [comment]
LGTM

**Pull request merged**